### DISCUSSION TITLE
Small docs improvement suggestion for HelmChartProps

### DISCUSSION THREAD

**Discussion by jameskelleher** [aws/aws-cdk#22013] [General]
Hello, I wanted to leave a comment that the [HelmChartProps documentation](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_eks.HelmChartProps.html) could use some clarification for how values work. I just spent a good amount of time trying to figure out why my values did not seem to be getting passed when they were like this:

```typescript
values: {
  'installCRDs': true,
  'webhook.port': 9443
}
```

Finally realized it's because they are expected to be formatted like this:

```typescript
values: {
  installCRDs: true,
  webhook: {
    port: 9443
  }
}
```
Would have saved me a lot of time had the docs mentioned that nested values should use nested dictionaries instead of using dot notation.

**Comment by pahud** [comment]
Yes that would be very helpful. Are you able to create a PR to clarify this in the [README](https://github.com/aws/aws-cdk/tree/main/packages/%40aws-cdk/aws-eks#helm-charts) for that?

**Reply by jameskelleher** [reply]
@pahud Yea! I can definitely take care of that :)

**Comment by jameskelleher** [comment]
https://github.com/aws/aws-cdk/pull/22142
